src/js/auth.js

```diff
@@ -545,6 +543,30 @@ const isAuth = async (req, res, next) => {
         return next();
       }
 
+      matcher = url.match(/\/api\/organizations\.*/);
```

Comment:
I'm not sure what this is doing that isn't already handled by the organization user url matcher on line 618 below.
If we're going to allow an organization user to query organizations, they will need to be restricted to querying/updating their own org and child orgs.

Reply:
Adding this matcher was a compromise because the client wasn't passing along the org id in its request ("organizations?filter[where][type]=O") and the original matcher pattern was looking for it, but neither the organization.controller nor the planterOrganization.controller have an endpoint for that pattern, "organization/organizationId/organizations".

So the original matcher works for permissions but the request returns a 404 because the endpoint doesn't exist.

Alternatives: 
We can add an endpoint for "organization/organizationId/organizations" to one of the controllers or pass along the org Id in a filter to the organization.controller

If we want to require all logins to have a orgId then we might update the organization.controller endpoints